docs/server/push-duty.rst

```diff
@@ -144,9 +159,12 @@ deployed revision and tag e.g:
  * `Addons Frontend (stage) <https://addons.allizom.org/__frontend_version__>`_
  * `Addons Code Manager (stage) <https://code.addons.allizom.org/__version__>`_
 
+Note that for Extension Workshop, pushing a tag to ``upstream`` with a name including
+the ``-stage`` suffix, will automatically deploy the tag to stage. You should manually
+verify Extension Workshop on stage after the push has completed.
```

Comment:
```suggestion
verify Extension Workshop on stage after the push has completed. Visit https://extensionworkshop.allizom.org and view any pages that have been changed since the last push to verify they exist and are rendering properly.
```